mqtt/mqtt-edgehub/src/command/handler.rs

```diff
@@ -70,14 +73,17 @@ impl CommandHandler {
             Duration::from_secs(60),
         );
 
-        let subscribe_topics = commands.keys().cloned().collect::<Vec<_>>();
-        subscribe(&mut client, subscribe_topics).await?;
-
-        Ok(CommandHandler {
-            broker_handle,
+        CommandHandler {
             client,
-            commands,
-        })
+            commands: HashMap::new(),
+        }
+    }
+
+    // TODO refactor and move it inside the [`run`] method
```

Comment:
What does the refactor entail? Why can't it be moved in as is?

Reply:
As per discussion with @and-rewsmith. That's what we agreed to do next to avoid sidecars startup order.